packages/e2e-tests/plugins/inner-blocks-templates/index.js

```diff
@@ -24,55 +24,106 @@
 		],
 	];
 
+	var TEMPLATE_TWO_PARAGRAPHS = [
+		[
+			'core/paragraph',
+			{
+				fontSize: 'large',
+				content: 'One',
+			},
+		],
+		[
+			'core/paragraph',
+			{
+				fontSize: 'large',
+				content: 'Two',
+			},
+		],
+	];
+
 	var save = function () {
-		return el( InnerBlocks.Content );
+		return el(InnerBlocks.Content);
 	};
 
-	registerBlockType( 'test/test-inner-blocks-no-locking', {
+	registerBlockType('test/test-inner-blocks-no-locking', {
```

Comment:
For some reason prettier isn't working on these files. I tried the `npm run format-js` command as well.

Reply:
I don't think this is a major blocker for the PR.

I had a brief look and couldn't see why this is happening.

Reply:
#28033 has background and restores formatting and linting.